plugins/axe.ts

```diff
@@ -0,0 +1,6 @@
+import Vue from 'vue';
+import VueAxe from 'vue-axe';
+
+if (process.env.NODE_ENV === 'development') {
+  Vue.use(VueAxe);
+}
```

Comment:
The docs use `require` after the if statement. Should we be doing that here?
```
import Vue from 'vue'

if (process.env.NODE_ENV === 'development') {
  const VueAxe = require('vue-axe').default
  Vue.use(VueAxe)
}
```

Or
```
import Vue from 'vue'

if (process.env.NODE_ENV === 'development') {
 import('vue-axe').then(({default: VueAxe}) => {
   Vue.use(VueAxe)
 })
  
}

```